flatliners/comparisonscore.py

```diff
@@ -0,0 +1,57 @@
+from .baseflatliner import BaseFlatliner
+
+
+class ComparisonScore(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+
+        self.score = dict()
+        self.clusters = dict()
+        self.versions = dict()
+
+
+    def on_next(self, x):
+        """ update l2 distance between cluster vector and baseline vector
+        """
+        # determine if entry is a version metric or if it is a cluster metric
+        is_version_record = list(x.keys())[0] == 'version'
+        is_cluster_record = list(x.keys())[0] == 'cluster'
+
+        # for version records, collect the average standard deviation value for
+        # each resource
+        if is_version_record:
+            self.set_version_std(self.versions, x)
```

Comment:
If you're passing in `self.versions` I would not make this method static. You could just pass in `x` and then access `self.versions` in the method itself.

In fact I find static methods not so useful anyway.

But let's keep it that way and do some refactoring maybe later

Reply:
Ok, sounds good. Yeah, I can change that on the next PR.